airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -129,10 +129,14 @@ def _group_logs_by_host(logs):
             key = getattr(log, 'host', 'default_host')
             grouped_logs[key].append(log)
 
-        # return items sorted by timestamp.
-        result = sorted(grouped_logs.items(), key=lambda kv: getattr(kv[1][0], 'message', '_'))
+        # return items sorted by asctime.
+        def sorter(log):
+            return getattr(log, 'asctime', '_')
 
-        return result
+        for host_logs in grouped_logs.values():
+            host_logs.sort(key=sorter)
```

Comment:
Is it not possible to make ElasticSearch sort things for us?

Reply:
I'm not very familiar with ElasticSearch, I'll try to research